src/src/app.cpp

```diff
@@ -730,6 +731,9 @@ bool CodeBlocksApp::OnInit()
             }
         }
 
+        if (!m_Batch)
+            Manager::Get()->GetUserVariableManager()->SetUI(new UserVarManagerGUI());
```

Comment:
I think this should be something like `SetUI(!m_Batch? new UserVarManagerGUI : new UserVarManagerGUIBatch);`